lib/Dialect/SV/Transforms/HWMemSimImpl.cpp

```diff
@@ -33,7 +35,27 @@ struct FirMemory {
   size_t readLatency;
   size_t writeLatency;
   size_t readUnderWrite;
+  size_t writeUnderWrite;
+  llvm::EquivalenceClasses<unsigned> writeSets;
 };
+
+class SymbolDigger {
```

Comment:
Plz give this a better name, and a doc comment.

Reply:
Removed due to the rework.